pkg/app/piped/controller/scheduler.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 var (
-	defaultDeploymentTimeout = time.Hour
+	defaultDeploymentTimeout = 6 * time.Hour
```

Comment:
nit, do we have docs that related to this default value 👀

Reply:
This will be included while fixing https://github.com/pipe-cd/pipe/issues/1373

Reply:
got it, thank you 👍